src/transformers/pipelines/audio_utils.py

```diff
@@ -70,6 +71,10 @@ def ffmpeg_microphone(
             The indentifier of the input device to be used by ffmpeg (i.e. ffmpeg's '-i' argument). If unset,
             the default input device will be used. See `https://www.ffmpeg.org/ffmpeg-devices.html#Input-Devices`
             for how to specify and list input devices.
+        background_arg (`str`, *optional*):
```

Comment:
"background_arg" isn't very descriptive. It would be better to just propogate `run_in_background` to this function too if there's only two possible options and then set the correct string within this function